cypress/features.md

```diff
@@ -16,7 +16,7 @@ The following features will be implemented. This gets done in three steps:
 
 ### User Account
 
-[Cucumber Features](./integration/user_account)
+[Cucumber Features](https://github.com/Human-Connection/Human-Connection/tree/master/cypress/integration/user_account)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-08T20:17:52Z" title="Monday, July 8th 2019, 10:17:52 pm +02:00">Jul 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -16,7 +16,7 @@ The following features will be implemented. This gets done in three steps:
 
 ### User Account
 
-[Cucumber Features](./integration/user_account)
+[Cucumber Features](https://github.com/Human-Connection/Human-Connection/tree/master/cypress/integration/user_account)
```


```suggestion
[Cucumber Features](./integration/user_account)
```
In this case it would be better to keep the relative path. Why? If you send in a PR and let's say you changed the referenced file and if a code reviewer would click on the link, she would be redirected to the `master` branch. You can try it out yourself by clicking on "Preview" and then click on the link.

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-07-09T10:16:08Z" title="Tuesday, July 9th 2019, 12:16:08 pm +02:00">Jul 9, 2019</time>_

---


You're right, that's not ideal. The reason I changed it is that the relative paths are not working on GitBook. For some reason they add a `/README.md` to the end of the link and since there are no `README`s in the `cypress/integrations` folder you land on a 404 page...

Any idea how to get this working for both use cases?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-09T13:18:25Z" title="Tuesday, July 9th 2019, 3:18:25 pm +02:00">Jul 9, 2019</time>_

---


No! In that case I'll approve this and merge. :+1: